examples/twittermap/web/public/javascripts/map/controllers.js

```diff
@@ -430,6 +435,22 @@ angular.module('cloudberry.map', ['leaflet-directive', 'cloudberry.common','clou
         colors: [ '#ffffff', '#92d1e1', '#4393c3', '#2166ac', '#f4a582', '#d6604d', '#b2182b'],
         sentimentColors: ['#ff0000', '#C0C0C0', '#00ff00']
       };
+      
+      if ($scope.polygons.statePolygons) {
+        $scope.map.removeLayer($scope.polygons.statePolygons);
+      }
+      if ($scope.polygons.countyPolygons) {
+        $scope.map.removeLayer($scope.polygons.countyPolygons);
+      }
+      if ($scope.polygons.cityPolygons) {
+        $scope.map.removeLayer($scope.polygons.cityPolygons);
+      }
+      if ($scope.polygons.stateUpperPolygons) {
+        $scope.map.removeLayer($scope.polygons.stateUpperPolygons);
+      }
+      if ($scope.polygons.countyUpperPolygons) {
+        $scope.map.removeLayer($scope.polygons.countyUpperPolygons);
+      }
```

Comment:
These lines that remove layers are duplicated in the cleanCountMap() function. From cleanCountMap() function, can you extract these common layer removal parts and create another function and use it in the both places?

Reply:
For these remove layers actions, I think the cleanCountMap() function already did. And every time the setInfoControlPointMap() is called, the cleanCountMap() must have been called previously. So I guess these lines are duplicate.

Reply:
These layers should be removed when we switch from pointmap to countmap, too.

Reply:
Teyu, why that's the case?

Reply:
Because right now pointmap still add those polygons to the map, but with lighter color.

Reply:
I added a new function removePolygonLayers() that will be used by both cleanCountMap() and cleanPointMap().